### PR TITLE
wording & tweaks to readme.md

### PR DESCRIPTION
**Pull request by Stillstellung**
- Made some minor clarifying changes to the **Weather** related information in readme.md
- Added line items for slew mode behavior to the will sync and wont sync sections

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -94,15 +94,17 @@ Notes:
...
+* Position/Rotation when changed by Slew Mode
 * Controls
 * Every functional switch
 * Avionics/FMCs
 * Autopilot
 * Time (on initial connection)
 
 ## What You Can Not Expect To Sync
-* Speed when weather is desynced (ISA/Temperature difference)
+* Speed when Weather is not the same across users (ISA/Temperature difference)
```
Comment (Sequal32) [inline Readme.md:105]: weather should be lowercased

File: Readme.md [hunk 2]
```diff
@@ -94,15 +94,17 @@ Notes:
 
 ## What You Can Expect to Sync
 * Position/Rotation
+* Position/Rotation when changed by Slew Mode
```
Comment (Sequal32) [inline Readme.md:97]: I don't think this is needed as changing position/rotation in slew mode is still the same as position/rotation on its own

**Review by Sequal32 — CHANGES_REQUESTED** [review]

**Review by Stillstellung — COMMENTED** [review]
@Sequal32 
sounds good to me on both accounts. 
mostly, was just surprised that slew mode synced, but it does make sense if your code is syncing positional data. not sure why I was surprised! hah

**Comment by Sequal32** [comment]
haha that's great - if you'd like to finish this up by making the changes then adding on another commit, I can approve and then merge it

**Pull request closed without merging**